labs/src/main/scala/org/scalalabs/basic/lab03/OptionExercise.scala

```diff
@@ -25,7 +25,11 @@ object OptionExercise01 {
    * - does not exist: 					"not existing"
    */
   def roomState(rooms: Map[Int, Option[String]], room: Int): String = {
-    error("Fix me")
+    if (!rooms.contains(room)) "not existing" else rooms(room) match {
```

Comment:
I'm trying to decide if I like this better or worse than just including the "not existing" check within the match statement. This seems like a nice way to avoid entering the match statement if it isn't necessary, but it looks sightly less elegant. I guess it depends on how much of tradeoff it would be to eliminate the if statement.